spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreter.scala

```diff
@@ -204,9 +164,6 @@ object MappingRuleInterpreter {
       }
       recoverFunction
     })
-
-    // Rethrow
-    validationResult.get
```

Comment:
Why is the rethrow suddenly not needed?

Reply:
I thought the code is not needed. But now rethinking, that it actually should be there... 🤔 Will come back...

Reply:
OK, I put it back. Changed the code a tiny bit to avoid the IDE's misleading hint.